platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -251,6 +254,13 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 		}
 	};
 
+    @Override
+    public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
+        for(int i = 0; i < singleton_count; i ++) {
+            singletons[i].onMainRequestPermissionsResult(requestCode, permissions, grantResults);
+        }
+    };
```

Comment:
It's asking you to use tab-based indentation here, not space-based indentation.

Reply:
Oh ok thanks for pointing that out @LikeLakers2 . I would have thought spaces were the standard.

Reply:
I mean, the rest of the file is using tabs, so...

Reply:
When I attmepted to use clang-format it attempted to convert the entire document to spaces... but then complained of 
```
YAML:47:31: error: unknown key 'BreakConstructorInitializers'
BreakConstructorInitializers: AfterColon
```